src/store/initialState.js

```diff
@@ -22,4 +22,18 @@ module.exports = {
       content: '',
     },
   },
+  singlePost: {
+    post: {
+      author: {},
+      slug: '',
+      userType: '',
+      description: '',
+      likesCount: 0,
+      createdAt: '',
+    },
+    relatedPosts: {
+      isEmpty: false,
+      feed: [],
+    },
+  },
```

Comment:
@dusmel Why did you add this?

Reply:
Because someone was using them without giving them a default value ( and made them required). it was complaining.